CIP-XXXX/cardano-babbage.json

```diff
@@ -0,0 +1,1826 @@
+        },
+        "validity_start_interval": {
+          "$ref": "cardano-babbage.json#/definitions/UInt64"
+        },
+        "withdrawals": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "key": {
+                "$ref": "cardano-babbage.json#/definitions/RewardAddress"
+              },
+              "value": {
+                "$ref": "cardano-babbage.json#/definitions/UInt64"
+              }
+            },
+            "additionalProperties": false
+          }
+        }
+      },
+      "required": [
+        "inputs",
+        "outputs",
+        "fee"
+      ],
+      "additionalProperties": false
+    },
+    "NetworkId": {
+      "title": "NetworkId",
+      "type": "string",
+      "enum": [
+        "testnet",
+        "mainnet"
+      ]
+    },
+    "PlutusData": {
+      "title": "PlutusData",
+      "discriminator": {
+        "propertyName": "tag"
+      },
+      "oneOf": [
+        {
+          "title": "PlutusDataList",
+          "type": "object",
+          "properties": {
+            "tag": {
+              "enum": [
+                "list"
+              ]
+            },
+            "contents": {
+              "type": "array",
+              "items": {
+                "$ref": "cardano-babbage.json#/definitions/PlutusData"
+              }
+            }
+          }
+        },
+        {
+          "title": "PlutusDataConstr",
+          "type": "object",
+          "properties": {
+            "tag": {
+              "enum": [
+                "constr"
+              ]
+            },
+            "alternative": {
+              "$ref": "cardano-babbage.json#/definitions/UInt64"
+            },
+            "data": {
+              "type": "array",
+              "items": {
+                "$ref": "cardano-babbage.json#/definitions/PlutusData"
+              }
+            }
+          }
+        },
+        {
+          "title": "PlutusDataMap",
+          "type": "object",
+          "properties": {
+            "tag": {
+              "enum": [
+                "map"
+              ]
+            },
+            "contents": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "key": {
+                    "$ref": "cardano-babbage.json#/definitions/PlutusData"
+                  },
+                  "value": {
+                    "$ref": "cardano-babbage.json#/definitions/PlutusData"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "title": "PlutusDataInteger",
+          "type": "object",
+          "properties": {
+            "tag": {
+              "type": "string",
+              "enum": [
+                "integer"
+              ]
+            },
+            "value": {
+              "$ref": "cardano-babbage.json#/definitions/Int128"
```

Comment:
My previous comment about slot and unit was wrong. But anyway plutus_data has datatype differ than int128. In the [cddl](https://github.com/IntersectMBO/cardano-ledger/blob/6d860294304749a767d99db92fab42f7c18d9af4/eras/babbage/impl/cddl-files/babbage.cddl#L308-L317) it can be a  bignum that is defined here https://datatracker.ietf.org/doc/html/rfc7049#section-2.4.2

Reply:
> In the [cddl](https://github.com/IntersectMBO/cardano-ledger/blob/6d860294304749a767d99db92fab42f7c18d9af4/eras/babbage/impl/cddl-files/babbage.cddl#L308-L317) it can be a bignum

Hmmm, 

```
big_int = int / big_uint / big_nint
big_uint = #6.2(bounded_bytes)
big_nint = #6.3(bounded_bytes)
```

```
bounded_bytes = bytes .size (0..64)
  ; the real bounded_bytes does not have this limit. it instead has a different
  ; limit which cannot be expressed in CDDL.
  ; The limit is as follows:
  ;  - bytes with a definite-length encoding are limited to size 0..64
  ;  - for bytes with an indefinite-length CBOR encoding, each chunk is
  ;    limited to size 0..64
  ;  ( reminder: in CBOR, the indefinite-length encoding of bytestrings
  ;    consists of a token #2.31 followed by a sequence of definite-length
  ;    encoded bytestrings and a stop code )
```

So, effectively we have a `BigInt` here (signed, long integer, arbitrary length). Thanks for catching this!